superset-frontend/src/components/ListViewCard/index.tsx

```diff
@@ -132,12 +134,23 @@ const CoverFooterRight = styled.div`
   text-overflow: ellipsis;
 `;
 
+const ThinSkeleton = styled(Skeleton)`
```

Comment:
If this is something you anticipate needing often, we can export the styled component from /src/common/components along with the rest of the AntD base components.

Reply:
moved to /src/common/components